src/components/Login.js

```diff
@@ -1,11 +1,123 @@
-import { Link } from 'react-router-dom'
+import { useState } from 'react'
+import {
+	ChakraProvider,
+	Box,
+	VStack,
+	Input,
+	Button,
+	Alert,
+	AlertIcon,
+	AlertTitle,
+	Center,
+} from '@chakra-ui/react'
+import useAuth from '../hooks/useAuth'
+import { useNavigate } from 'react-router-dom'
 
 const Login = () => {
-    return (
-        <>
-            <h1><Link to="dashboard">Login</Link></h1>
-        </>
-    )
+	const setAuth = useAuth().setAuth
+	const navigate = useNavigate()
+	const [formData, setFormdata] = useState({
+		facultyNumber: '',
+		password: '',
+	})
+
+	const [hasError, setError] = useState(false)
+	const [isInvalid, setIsInvalid] = useState(false)
```

Comment:
There is something wrong with the error mechanism - no error gets displayed when I try to log in with wrong credentials.